packages/fxa-payments-server/.storybook/config.js

```diff
@@ -0,0 +1,16 @@
+import { configure } from '@storybook/react';
+
+import '../src/index.scss';
+import '../src/App.scss';
+
+const reqFromComponents = require.context('../src/components', true, /\.stories.tsx?$/);
```

Comment:
Having a go at adding [Storybook](https://storybook.js.org/) to the mix here. Getting all the moving pieces in the stack set up well enough to exercise every UX state is kind of a nightmare. With Storybook, we can express all those states as "stories" with ready-baked React props and work on UX & styles from there.